airgun/widgets.py

```diff
@@ -21,6 +21,7 @@
 from widgetastic_patternfly import Kebab
 from widgetastic_patternfly import VerticalNavigation
 from widgetastic_patternfly4.ouia import BaseSelect
+from widgetastic_patternfly4.ouia import Button as PF4Button
```

Comment:
I see you have used it in PF4 Confirmation dialogue, so u might change it to `DialogButton` kinda naming!

Reply:
@jameerpathan111 I don't think all buttons are PF4 Buttons therefore we can't use `Button` name here because it's already in use.
and for changing it to `DialogButton` is not a good idea either because it has to be general name which can be 
used with other PF4 buttons already present on the page.